registrations/utils.py

```diff
@@ -151,16 +151,20 @@ def strip_trailing_zeroes_from_decimal(value: Decimal):
     return value.normalize()
 
 
-def move_first_waitlisted_to_attending(signup_or_signup_group):
-    registration = signup_or_signup_group.registration
-
-    if (
+def move_n_waitlisted_to_attending(registration, n_count: int):
```

Comment:
Same here, I'd prefere something like:
```suggestion
def move_waitlisted_to_attending(registration, count: int):
    """Changes given number of wait-listed attendees in registration to be attending."""
```

Reply:
Changed as suggested (same applies to all occurrences of this function and the `get_waitlisted` method).